code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -34,6 +34,7 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	item_state = "electronic"
 	w_class = W_CLASS_TINY
 	flags = FPRINT
+	origin_tech = Tc_POWERSTORAGE + "=2;" + Tc_BLUESPACE + "=1;" + Tc_PROGRAMMING  + "=4"
```

Comment:
The PDA printer exists for a reason

Reply:
So does the general fabricator

Reply:
More like the PDA Terminal exists for a reason 😕